Tests/MapboxNavigationTests/ManeuverViewSnapshotTests.swift

```diff
@@ -60,18 +68,19 @@ class ManeuverViewSnapshotTests: TestCase {
     }
 
     func testLeftUTurn() {
-        maneuverView.drivingSide = .right
         maneuverView.visualInstruction = maneuverInstruction(.turn, .uTurn)
-        let image = UIImage(view: maneuverView)!
-        let flipped = UIImage(cgImage: image.cgImage!, scale: UIScreen.main.scale, orientation: .upMirrored)
-        let imageView = UIImageView(image: flipped)
-        assertImageSnapshot(matching: imageView, as: .image(precision: 0.95))
+        maneuverView.drivingSide = .right
+        
+        assertImageSnapshot(matching: UIImageView(image: maneuverView.imageRepresentation),
+                            as: .image(precision: 0.95))
     }
 
     func testRightUTurn() {
-        maneuverView.drivingSide = .left
         maneuverView.visualInstruction = maneuverInstruction(.turn, .uTurn)
-        assertImageSnapshot(matching: maneuverView.layer, as: .image(precision: 0.95))
+        maneuverView.drivingSide = .left
+
+        assertImageSnapshot(matching: UIImageView(image: maneuverView.imageRepresentation),
```

Comment:
Using `UIView.imageRepresentation` makes sure to verify whether flipping transformations were applied. This is relevant for right/left driving side roads.